site/content/extend/plugins/best-practices.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Example Plugins"
+subsection: Plugins (Beta)
+weight: 50
+---
+
+See here for [server-specific best practices for plugins](/extend/plugins/server/best-practices/). Webapp-specific best practices are incoming.
+
+## How can I review the entire code base of a plugin?
+
+Carry out the following steps:
+
+1. Take a backup of project directory
+2. Create a dummy-master branch with no code:
+
+   ```
+   git checkout --orphan dummy-master #create the dummy master branch
+   git rm -rf . #remove everything from this branch
+   git push origin dummy-master
+   ```
+   
+3. Create a dummy-feature branch from dummy-master:
+
+   ```
+   git checkout -b dummy-feature
```

Comment:
Maybe call it `review` instead?

Reply:
You mean `dummy-review`?

Reply:
No, just `review`. The whole point of this process is the review the plugin code. Hence, `review` seams to be the best branch name to me.

Reply:
`dummy-XXX` helps make it clear all the review work is done against a dummy branch and hence changes won't affect master. 1/5

Reply:
If you are 1/5, I'm fine with `dummy-review`.